src/Nest/CommonAbstractions/ConnectionSettings/ConnectionSettingsBase.cs

```diff
@@ -161,9 +161,10 @@ public TConnectionSettings DefaultTypeNameInferrer(Func<Type, string> typeNameIn
 		}
 
         /// <summary>
-        /// Specify the default index names for a given POCO type. 
+        /// Specify the default index names for a given POCO type.
         /// Takes precedence over the global <see cref="DefaultIndex"/>
         /// </summary>
+        [Obsolete("Removed in 6.0.")]
```

Comment:
I'm inclined not to `Obsolete` this as it is likely to be noisy for a lot of users, since it is a commonly used method. Perhaps an XML comment instead?

Reply:
Moved to ///remarks